docs/tutorial.md

```diff
@@ -0,0 +1,369 @@
+            return UpdateControl.updateStatusSubResource(resource);
+        }
+```
+
+### createMemcachedDeployment
+
+Thie method simply creates the Deployment as specified below.
+
+```
+    private Deployment createMemcachedDeployment(Memcached m) {
+        return new DeploymentBuilder()
+            .withMetadata(
+                new ObjectMetaBuilder()
+                    .withName(m.getMetadata().getName())
+                    .withNamespace(m.getMetadata().getNamespace())
+                    .withOwnerReferences(
+                        new OwnerReferenceBuilder()
+                            .withApiVersion("v1alpha1")
+                            .withKind("Memcached")
+                            .withName(m.getMetadata().getName())
+                            .withUid(m.getMetadata().getUid())
+                            .build())
+                    .build())
+            .withSpec(
+                new DeploymentSpecBuilder()
+                    .withReplicas(m.getSpec().getSize())
+                    .withSelector(
+                        new LabelSelectorBuilder().withMatchLabels(labelsForMemcached(m)).build())
+                    .withTemplate(
+                        new PodTemplateSpecBuilder()
+                            .withMetadata(
+                                new ObjectMetaBuilder().withLabels(labelsForMemcached(m)).build())
+                            .withSpec(
+                                new PodSpecBuilder()
+                                    .withContainers(
+                                        new ContainerBuilder()
+                                            .withImage("memcached:1.4.36-alpine")
+                                            .withName("memcached")
+                                            .withCommand("memcached", "-m=64", "-o", "modern", "-v")
+                                            .withPorts(
+                                                new ContainerPortBuilder()
+                                                    .withContainerPort(11211)
+                                                    .withName("memcached")
+                                                    .build())
+                                            .build())
+                                    .build())
+                            .build())
+                    .build())
+            .build();
+    }
+```
+
+### deleteResource
+
+The `deleteResource` method is an implemented method. The deletion part will be taken care of by the Java Operator SDK  library, that's why it is empty.
+
+## Run the Operator Locally
+
+### 1. Run locally outside the cluster
+
+The following steps will show how to run your operator locally.
+
+Compile your operator with the below command
+
+`mvn clean install`
+
+It will create `.jar` file for your operator. You can go to the `target` folder and then to `quarkus-app`.
+
+Now, run the `jar` file using the below command.
+
+`java -jar quarkus-run.jar`
```

Comment:
Instead of telling them to go to the target folder, I would just change the java command to have the full path.
```suggestion
It will create a `.jar` file for your operator in `target/quarkus-app`.  Now, run the `jar` file using the below command.

`java -jar target/quarkus-app/quarkus-run.jar`
```

Reply:
Modified.